pages/common/nu.md

```diff
@@ -7,18 +7,18 @@
 
 `nu`
 
-- Execute a command and then exit:
+- Execute commands:
 
-`nu --commands "{{command}}"`
+`nu --commands "echo 'nu is executed'"`
 
 - Execute a script:
 
-`nu {{path/to/script.nu}}`
+`nu "{{path/to/script.nu}}"`
```

Comment:
What will happen, if we do not add quotes?

Reply:
Script name will be splitted on IFS if word splitting is on. That's why I prefer to use quotes, not to encounter to this issue.